docs/snippets/common/before-each-in-meta-mock-date.js.mdx

```diff
@@ -0,0 +1,26 @@
+```ts
```

Comment:
Missed one here. when you're able can you adjust this so that it aligns with the file? It shouldn't be `ts` but ´js`, and the same for comment referencing the filename.